src/Store/Actions/Actions.js

```diff
@@ -20,6 +20,7 @@ export const fetchAffectedSystemsByCVE = (cve, apiProps) => ({
 
 export const fetchSystems = options => ({
     type: ActionTypes.FETCH_SYSTEMS,
+    meta: new Date(),
```

Comment:
This might be a bit confusing, I think this would be better:
`meta: { timestamp: new Date() }`